last_task/server/server.go

```diff
@@ -0,0 +1,206 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+
+	_ "github.com/lib/pq"
+)
+
+type DB struct {
+	Conn *sql.DB
+}
+
+const (
```

Comment:
Some sensitive data, like passwords for DB or API keys, should be passes through env vars or config files. Please read about environment variables.